dali/test/cfitsio_test.cc

```diff
@@ -27,13 +27,17 @@ class CfitsioTest : public ::testing::Test {
   char filename[40];
   fitsfile *fptr;
 
-  virtual void SetUp() {
+  void SetUp() final {
     status = 0;
-    remove("testprog.fit");
+    if (remove("testprog.fit") != 0) {
```

Comment:
to be on the safe side, this file should proably live in a temporary dir

Reply:
True, but out of scope.